terraform/constants.go

```diff
@@ -1,5 +1,12 @@
 package terraform
 
 const (
-	TF = ".tf"
+	TF          = ".tf"
+	TFVARS      = ".tfvars"
+	AUTO_TFVARS = ".auto.tfvars"
+
+	DEFAULT_TFVARS = "terraform.tfvars"
 )
+
+var VALID_VARIABLE_FILES = [...]string{TF, AUTO_TFVARS}
```

Comment:
Why `DEFAULT_TFVARS` is not part of `VALID_VARIABLE_FILES`?

Reply:
Initially I had a different check for `DEFAULT_TFVARS` but now it would look like it's the same as the ones for `VALID_VARIABLE_FILES`. 

However, after [your comment here](https://github.com/snyk/snyk-iac-parsers/pull/12#discussion_r810589116) I will change this logic so for example **any** file ending in `.tf`. or `.auto.tfvars` is okay but for `DEFAULT_TFVARS` we need a file either directly named `terraform.tfvars` or ending in `/terraform.tfvars` (because the `fileName` is actually the path to the file.